analysis/org.eclipse.tracecompass.analysis.profiling.core/META-INF/MANIFEST.MF

```diff
@@ -42,12 +42,12 @@ Export-Package: org.eclipse.tracecompass.analysis.profiling.core.base;uses:="org
  org.eclipse.tracecompass.internal.analysis.profiling.core.instrumented;x-friends:="org.eclipse.tracecompass.analysis.profiling.core.tests,org.eclipse.tracecompass.analysis.profiling.ui",
  org.eclipse.tracecompass.internal.analysis.profiling.core.model;x-friends:="org.eclipse.tracecompass.analysis.profiling.core.tests",
  org.eclipse.tracecompass.internal.analysis.profiling.core.tree;x-friends:="org.eclipse.tracecompass.analysis.profiling.core.tests,org.eclipse.tracecompass.analysis.profiling.ui",
- org.eclipse.tracecompass.internal.provisional.analysis.profiling.core.base;x-friends:="org.eclipse.tracecompass.analysis.profiling.core.tests,org.eclipse.tracecompass.analysis.profiling.ui",
- org.eclipse.tracecompass.internal.provisional.analysis.profiling.core.callgraph;x-friends:="org.eclipse.tracecompass.analysis.profiling.core.tests",
- org.eclipse.tracecompass.internal.provisional.analysis.profiling.core.callstack;x-friends:="org.eclipse.tracecompass.analysis.profiling.ui,org.eclipse.tracecompass.analysis.profiling.core.tests",
- org.eclipse.tracecompass.internal.provisional.analysis.profiling.core.instrumented;x-friends:="org.eclipse.tracecompass.analysis.profiling.core.tests",
- org.eclipse.tracecompass.internal.provisional.analysis.profiling.core.model;x-friends:="org.eclipse.tracecompass.lttng2.ust.core,org.eclipse.tracecompass.analysis.profiling.core.tests",
- org.eclipse.tracecompass.internal.provisional.analysis.profiling.core.tree;x-friends:="org.eclipse.tracecompass.analysis.profiling.ui,org.eclipse.tracecompass.analysis.profiling.core.tests"
+ org.eclipse.tracecompass.internal.provisional.analysis.profiling.core.base,
```

Comment:
provisional APIs are APIs that are proposed and are supposed to be made public in the future. However, they still need to be marked as internal in the MANIFEST.MF (x-internal:=true or x-friends if there are any). If this PR is to make the APIs public, then the package name cannot contain provisional.

This applies for all relevant changed lines in the MANIFEST.MF files.